.github/workflows/pytest.yml

```
@@ -0,0 +1,34 @@
+name: Django CI with Docker Compose
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
```

```suggestion
on:
  workflow_dispatch:
  push:
      branches:
      - main
  pull_request:
```

we also like `workflow_dispatch` because it allows us to manually trigger. We also need it to run on every pull request and only run on all pushes for only the main branch